### ISSUE TITLE
Is the training script for knowledge distillation available?

### ISSUE THREAD

**Issue by zjysteven** [goldblum/AdversariallyRobustDistillation#6]
Hi,

Thanks for making the code public! I'm interested in replicating the standard knowledge distillation performance from a robust ResNet18 to a MobileNetV2 (with only clean images), which according to Table 3 can give 76.86% (clean) and 38.21% (robust) accuracies. However, my reproduced version only yields 82.88% (clean) and 11.60% (robust) accuracies. The only difference between mine and your experiment that I'm aware of could be 1) the difference in the teacher model; 2) some different hyperparameters, e.g. batch size, training epochs, learning rate schedule. However, I don't think the above differences can affect that much. Do you have any thoughts on this? And it would be appreciated if you can also release your training script for standard knowledge distillation.

Thanks!
